src/ExtensionsSample/Program.cs

```diff
@@ -66,7 +69,8 @@ public static void Main(string[] args)
                 typeof(SendGridSamples),
                 typeof(TableSamples),
                 typeof(TimerSamples),
-                typeof(WebHookSamples));
+                typeof(WebHookSamples),
+                typeof(ApiHubSamples));
```

Comment:
I'd rather you left this the way it was - when I merged your stuff last week, I moved the ApiHub samples to the same model that the EasyTables and other samples use. See both the sample files - they have doc on how to enable them. The idea is that if/when people want to run them, they can just add this line. This makes the samples much easier to run for people - they don't have to configure the world to get them running.